materialize-bigquery/bigquery.go

```diff
@@ -16,7 +17,10 @@ import (
 	"google.golang.org/api/option"
 )
 
-// Config represents the endpoint configuration for BigQuery.
+var featureFlagDefaults = map[string]bool{
+	"objects_and_arrays_as_json": false,
```

Comment:
The name is fairly self-explanatory in this case, but I've been trying to maintain a convention of describing the impact of a flag here in the defaults map.

Reply:
That's a good idea, I added a comment here to explain what the flag does.